src/Codec/JVM/Attr.hs

```diff
@@ -35,6 +35,7 @@ data Attr
     , codeAttrs :: [Attr] }
   | AStackMapTable [(Offset, StackMapFrame)]
   | AInnerClasses InnerClassMap
+  | ASignature Text
```

Comment:
@puffnfresh @rahulmutt  I hope this doesn't cause an issue with this https://github.com/rahulmutt/codec-jvm/blob/59c8c9a405e465ff6ccbad4d26b3240b800b4b62/src/Codec/JVM/Attr.hs#L49